### PR TITLE
Added contents to Readme.md

### PR DESCRIPTION
**Pull request by arjunraghunandanan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+* * Get ID 
+* * Get Password
+* * Refresh Data
+* * Subject wise attendence 
+
+
+# MASTER Repository 
+
+you can find the original repository in the link
```
Comment (subins2000) [inline README.md]: No need of this

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
-# vaasu
\ No newline at end of file
+# vaasu
+
+This Repository holds the code for Vaasu Project
+It is an Open source Project currently bring developed for Vidya Academy of Science and Technology by FossersVAST
```
Comment (subins2000) [inline README.md]: typo "being"

FOSSersVAST (notice the letter capitalizations)

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+
+The attendence data is will be as recieved from the student's college ERP
```
Comment (subins2000) [inline README.md]: typos

**Review by subins2000 — CHANGES_REQUESTED** [review]

**Comment by abspython** [comment]
I think we should complete the README after finishing the project.

**Comment by subins2000** [comment]
@abspython We can have a todo list or something in the readme for tracking maybe ?

**Pull request merged**